messages/whitespace-found.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the plugin "<%- pluginName %>". It looks like there is whitespace in the plugin name.
```

Comment:
I think we can be a bit more instructive here. Maybe:

> ESLint couldn't load the plugin "<%- pluginName %>" because there is whitespace in the name. Please check your configuration and remove all whitespace from the plugin name.